servers/tests/simulnet.rs

```diff
@@ -248,8 +259,14 @@ fn simulate_full_sync() {
 	let s1_header = s1.chain.head_header().unwrap();
 
 	// Wait for s2 to sync up to and including the header from s1.
+	let mut time_spent = 0;
 	while s2.head().height < s1_header.height {
 		thread::sleep(time::Duration::from_millis(1_000));
+		time_spent += 1;
+		if time_spent >= 60 {
+			println!("sync fail. s2.head().height: {}, s1_header.height: {}", s2.head().height, s1_header.height);
```

Comment:
Should we fail the test here?

Reply:
`assert_eq!(s1_header, s2_header);` on line 274 will do that, no need to duplicate here.